.github/workflows/artifacts.yml

```diff
@@ -0,0 +1,83 @@
+name: Artifacts generation and upload
+on:
+  workflow_run:
+    workflows: ["Mattermost Build"]
+    types:
+      - completed
+jobs:
+  upload-s3:
+    name: cd/Upload artifacts to S3
+    runs-on: ubuntu-22.04
+    env:
+      REPO_NAME: ${{ github.event.repository.name }}
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success'
+    steps:
+      - name: cd/Configure AWS
+        uses: aws-actions/configure-aws-credentials@07c2f971bac433df982ccc261983ae443861db49 # v1-node16
+        with:
+          aws-region: us-east-1
+          aws-access-key-id: ${{ secrets.PR_BUILDS_BUCKET_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.PR_BUILDS_BUCKET_AWS_SECRET_ACCESS_KEY }}
+      - name: cd/Download artifacts
+        uses: dawidd6/action-download-artifact@0c49384d39ceb023b8040f480a25596fd6cf441b # v2.26.0
+        with:
+          workflow_conclusion: success
+          name: server-dist-artifact
```

Comment:
I would prefer to add the `path` variable here. 
Currently yes the github workspace does not have any other files but in the future we might need to add different files . 
Each file should have it's own repo

Reply:
Chosen to remove the path, for shake of simplicity. 
It should be easy to include the path. Will rebase and push a fix.

Reply:
I think it would be better

Reply:
Fixed with https://github.com/mattermost/mattermost-server/compare/a6f7e73e5165d1261f4e55dd04b50e7a67cecfc0..4a6c1d500f8fa1ff3a935f4a5b52be3f23df19c5